watches.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-- version: v1alpha1
+- version: v1beta1
```

Comment:
We don't watch multiple versions because when you create a v1alpha1 object it is converted to v1beta1 (not the `spec` though). We just do checks in Ansible looking for the old parameters and drop into a conversion routine. In the future we could look at updating/rewriting the current CR possibly.

Existing objects are also converted over to the new API version automatically, so there is no need to watch multiple objects and have separate entry points.